views/appView.js

```diff
@@ -0,0 +1,55 @@
+var AppView = Backbone.View.extend({
+  el: 'body',
+
+  collection: VideosCollection,
+
+  events: {
+    'click #search-submit': 'search',
+  },
+
+  initialize: function () {
+    this.listenTo(this.model, 'change', this.renderVideo);
+  },
+
+  // search: function () {
```

Comment:
This is close... line 17 should have been telling the model the search term you got in 16 and then having it fetch and then you have this View class listen for the collection change and re-render.